<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-58213208-1', 'auto');
  ga('send', 'pageview');

</script>

  <head>
  <meta charset="UTF-8">
  <link href="http://gmpg.org/xfn/11" rel="profile">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Data &middot; Amphiencode
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/atom+xml" title="Amphiencode" href="/atom.xml">
</head>


  <body>

    <div class="container content">
      <div class="masthead">
        <h3 class="masthead-title">
          <a href="/" title="Home">Amphiencode</a>

          
              &nbsp;&nbsp;&nbsp;<small><a href="/Project/">Project</a></small>
          
              &nbsp;&nbsp;&nbsp;<small><a href="/Data/">Data</a></small>
          
              &nbsp;&nbsp;&nbsp;<small><a href="/Contacts/">Contacts</a></small>
          

        </h3>
      </div>

      <article class="page">
  <h1 class="page-title">Data</h1>
  <div class="message">
  <b><u>Disclaimer:</u> This dataset is made available to the members of the European amphioxus genome consortium as collaborative material. These datasets are not to be used for whole genome analyses unless otherwise agreed for, and not to be redistributed to anybody outside the consortium without prior arrangements. The functional genomics data (ATAC-seq, Chip-seq, Methylation, RNA-seq) included in the browser are provided as a courtesy by several participant labs and remain their property (see contacts).</b></div>

<h2 id="datasets">Datasets</h2>

<p>The genome of the European amphioxus <em>Branchiostoma lancealatum</em> was generated from 150x Illumina coverage sequenced by <a href="http://www.genoscope.cns.fr">Genoscope</a>. In the Bl71nemr assembly, haplotypes were reconcilied using haplomerger. </p>

<p>This assembly spans 495.3Mb (N50: 1.29Mb) split in 10,247 scaffolds with 4% of residual gaps. Masked regions are represented with lowercase characters (soft-masking); gaps in the assembly are represented with Ns.</p>

<p>The genome was annotated using both EVM/PASA pipeline and cufflinks/transdecoder. Both these annotation have their strength. Cufflinks provides a better representation of transcript diversity, limits and TSSs positions. Conversely, EVM generates more robust predicted proteins, especially in regard to frameshifts, etc… A final annotation with unified IDs (BL00000) was subsequently built by merging the two annotations with a suffix indicative of their origin (BL00000_evm0 or BL00000_cuff0). All three annotations are available as GTFs. </p>

<table>
  <thead>
    <tr>
      <th>File</th>
      <th>Dataset</th>
      <th>Format</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong><a href="https://www.dropbox.com/s/4bwtoi217oplwef/Bl71nemr.fa.gz?dl=1">Bl71nemr.fa</a></strong></td>
      <td>Genome assembly</td>
      <td>fasta</td>
    </tr>
    <tr>
      <td><strong><a href="https://www.dropbox.com/s/q5zdshvos94r0pn/Bla_annot_final.gtf.gz?dl=1">Bla_annot_final.gtf</a></strong></td>
      <td>Final merged annotation (EVM/PASA+cufflinks)</td>
      <td>GTF</td>
    </tr>
    <tr>
      <td><strong><a href="https://www.dropbox.com/s/j3ul5orz1wj0v66/Bla_annot_evm.gtf.gz?dl=1">Bla_annot_evm.gtf</a></strong></td>
      <td> EVM/PASA annotation </td>
      <td>GTF </td>
    </tr>
    <tr>
      <td><strong><a href="https://www.dropbox.com/s/dtiql5qpjcoi9y3/Bla_annot_cuff.gtf.gz?dl=1">Bla_annot_cuff.gtf</a></strong></td>
      <td>Cufflinks annotation</td>
      <td>GTF </td>
    </tr>
    <tr>
      <td><strong><a href="https://www.dropbox.com/s/a9zzzgqqy0fvskg/Bla_annot_final_refTranscripts.fa.gz?dl=1">Bla_annot_final_refTranscripts.fa</a></strong></td>
      <td>Reference transcripts*</td>
      <td>fasta</td>
    </tr>
    <tr>
      <td><strong><a href="https://www.dropbox.com/s/hoyt0byn3a9pohz/Bla_annot_final_refProteins.fa.gz?dl=1">Bla_annot_final_refProteins.fa</a></strong></td>
      <td>Reference proteins**</td>
      <td>fasta</td>
    </tr>
  </tbody>
</table>

<p><em>(*) most expressed transcript for each locus; (**) longest protein for each locus</em></p>

<h2 id="blast-server">Blast server</h2>

<p>A <strong><a href="http://zoo-animalia.zoo.ox.ac.uk/blastseq04">blast server</a></strong> makes it possible to search the assembly, the reference transcripts and proteins. </p>

<h2 id="genome-browser">Genome browser</h2>

<p>The European amphioxus genome assembly and annotation are available as a <a href="https://genome.ucsc.edu/goldenPath/help/hgTrackHubHelp.html">UCSC track hub</a>. The assembly should be searchable using BLAT. To enable access, you can either: </p>

<ul>
  <li>Use the <a href="http://genome-euro.ucsc.edu/cgi-bin/hgTracks?hgS_doOtherUser=submit&amp;hgS_otherUserName=fmarletaz&amp;hgS_otherUserSessionName=basic_braLan1">following UCSC session</a> </li>
  <li>set up the hub by adding the hub address <code class="highlighter-rouge">http://zoo-animalia.zoo.ox.ac.uk/braLan/hub.txt</code> to <code class="highlighter-rouge">My Data &gt; Track Hubs &gt; My hubs</code> in UCSC menu. </li>
</ul>


</article>


  

      <div class="footer">
        <p>
          &copy; 2015. All rights reserved.
        </p>
      </div>
    </div>

  </body>
</html>